src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -623,7 +641,38 @@ assertPost env premem preregs = do
   liftIO $ C.addAssertion sym . C.LabeledPred correctStack . C.SimError W4.initializationLoc
     $ C.AssertFailureSimError "Stack not preserved" ""
 
-  -- TODO: Match return value in RAX
+  pointsToMatches <- forM (ms ^. MS.csPostState . MS.csPointsTos)
```

Comment:
I think points-to matches go after the return match, in case the function returns a pointer